docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+services:
+  db:
+    image: postgres:12
+    environment:
+      - POSTGRES_DB=${DATABASE_NAME:-navody_slovensko_digital_development}
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    expose:
+      - 5432/tcp
+    networks:
+      - navody
+    ports:
+      - 5432:5432
+    restart: unless-stopped
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+  app:
+    image: navody
+    build: .
+    command: "tail -f /dev/null"
+    depends_on:
+      - db
+    environment:
```

Comment:
miesto tohto by som ocakaval skor:

```
    env_file:
      - .env

Reply:
Prvotny umysel bol ten, aby novy developer na projekte musel zadavat minimum prikazov. Takto staci naklonovat repo, `docker-compose up --build -d` a rovno sa vie rails pripojit z `web` kontajnera do `db`.

`.env` file sa autoloaduje automaticky, takze ho netreba explicitne definovat pomocou `env_file`.

Dobre, ze si to ale spomenul, lebo `environment` v compose file ma prioritu pred `.env`, preto sa mozeme rozhodnut:
1. vymazme cely `environment` blok, pridajme `.env.example` (obsahujuci dostupne premenne) a upravme README, aby novy developer vedel, ze si ma nasetupovat `.env`, aby si prostredie rozbehal
2. nechajme to v tejto podobe pre out-of-box magic experience :-)